source/main/gui/panels/GUI_MainSelector.cpp

```diff
@@ -887,7 +888,17 @@ void CLASS::SetPreviewImage(Ogre::String texture)
 void CLASS::ResizePreviewImage()
 {
 	MyGUI::IntSize imgSize(0, 0);
-	Ogre::TexturePtr t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	Ogre::TexturePtr t;
+	try
+	{
+		t = Ogre::TextureManager::getSingleton().load(m_preview_image_texture, Ogre::ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME);
+	}
+	catch (...)
+	{
+		Ogre::LogManager::getSingleton().stream() << "[RoR|SelectorGUI] Failed to load preview image: " << m_preview_image_texture;
+		m_preview_image_texture.clear();
+		m_Preview->setVisible(false);
+	}
```

Comment:
Exception shouldn't pass un-noticed. The player (mod author?) should know the image is corrupted.

`m_Preview->setVisible(false);` is appropriate - without it, the exception will be thrown on every UpdateCoord event. No point in that.

`m_preview_image_texture.clear();` is not technically necessary, but feels like an appropriate cleanup.